.gitignore

```diff
@@ -118,9 +118,8 @@ dist
 # vuepress build output
 .vuepress/dist
 
-# vuepress v2.x temp and cache directory
+# vuepress v2.x temp
 .temp
-.cache
```

Comment:
Any specific reason to remove these from .gitignore? 

I can't see why would we want a folder named `.cache` outside of the ignored directories.

Reply:
it was repeated, there's another on line 101